cmd/run.go

```diff
@@ -414,7 +414,65 @@ func isSkipScript(hooksGroup, executableName string) bool {
 
 func isSkipCommand(hooksGroup, executableName string) bool {
 	key := strings.Join([]string{hooksGroup, commandsConfigKey, executableName, skipConfigKey}, ".")
-	return viper.GetBool(key)
+
+	value := viper.Get(key)
+
+	switch typedValue := value.(type) {
+	case bool:
+		/*
+			pre-push:
+				commands:
+					packages-audit:
+						skip: true
+		*/
+		return typedValue
+	case string:
+		/*
+			pre-push:
+				commands:
+					packages-audit:
+						skip: merge
+		*/
+		return isSkippedGitState(typedValue)
+	case []interface{}:
+		/*
+			pre-push:
+				commands:
+					packages-audit:
+						skip:
+							- merge
+							- rebase
+		*/
+		for _, gitState := range typedValue {
+			if isSkippedGitState(gitState.(string)) {
+				return true
+			}
+		}
+	}
+
+	return false
+}
+
+func isSkippedGitState(state string) bool {
+	return state == "merge" && isMergeInProgress() || state == "rebase" && isRebaseInProgress()
+}
+
+func isMergeInProgress() bool {
+	if _, err := os.Stat(filepath.Join(getRootPath(), ".git", "MERGE_HEAD")); os.IsNotExist(err) {
```

Comment:
Inside git worktrees, `getRootPath(), ".git"` will be a file, not directory! Also every worktree has its own git directory placed _inside_ `.git` directory of the main worktree :exploding_head:

Reply:
Fixed

Reply:
And fixed one more time for worktrees (pushed commit right into your pull request, sorry!)